app/src/main/res/layout/activity_description.xml

```diff
@@ -23,201 +23,232 @@
-        app:layout_constraintTop_toBottomOf="@+id/activity_text1_description" />
-
-    <TextView
-        android:id="@+id/activity_text3_description"
-        android:layout_width="35dp"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="20dp"
-        android:fontFamily="@font/armata"
-        android:layout_marginTop="30dp"
-        android:text="At"
-        app:layout_constraintStart_toEndOf="@+id/activity_duration_description"
-        app:layout_constraintTop_toBottomOf="@+id/activity_text1_description" />
-
-    <TextView
-        android:id="@+id/activity_address_description"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:fontFamily="@font/armata"
-        android:layout_marginStart="20dp"
-        android:layout_marginTop="30dp"
-        android:text="Address"
-        app:layout_constraintStart_toEndOf="@+id/activity_text3_description"
-        app:layout_constraintTop_toBottomOf="@+id/activity_date_description" />
-
-    <TextView
-        android:id="@+id/activity_text4_description"
-        android:layout_width="30dp"
-        android:layout_height="wrap_content"
-        android:fontFamily="@font/armata"
-        android:layout_marginStart="20dp"
-        android:layout_marginTop="30dp"
-        android:text="By"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/activity_text2_description" />
-
-    <TextView
-        android:id="@+id/activity_organisator_description"
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        android:fontFamily="@font/armata"
-        android:layout_marginStart="20dp"
-        android:layout_marginTop="30dp"
-        android:text="Organisator"
-        app:layout_constraintStart_toEndOf="@+id/activity_text4_description"
-        app:layout_constraintTop_toBottomOf="@+id/activity_duration_description" />
-
-    <TextView
-        android:id="@+id/activity_text5_description"
-        android:layout_width="30dp"
-        android:layout_height="wrap_content"
-        android:fontFamily="@font/armata"
-        android:layout_marginStart="20dp"
-        android:layout_marginTop="30dp"
-        android:text="For"
-        app:layout_constraintStart_toEndOf="@+id/activity_organisator_description"
-        app:layout_constraintTop_toBottomOf="@+id/activity_address_description" />
-
-
-    <TextView
-        android:id="@+id/activity_number_description"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:fontFamily="@font/armata"
-        android:layout_marginStart="20dp"
-        android:layout_marginTop="30dp"
-        android:text="Number participant"
-        app:layout_constraintStart_toEndOf="@+id/activity_text5_description"
-        app:layout_constraintTop_toBottomOf="@+id/activity_address_description" />
+        <ScrollView
```

Comment:
Nice to have those nested scroll views in our ActivityDescriptionActivity (especially if we further add additional information about the sports activity here in the future!).